### ISSUE TITLE
Add title to confirm-email route

### ISSUE THREAD

**Issue by liamsanft** [liamsanft/findyourrave#12]
**Describe the bug**
Currently the /auth/confirm-email route doesn't have a title. This needs to be fixed for SEO and accessibility.

**To Reproduce**
1. run ```yarn dev```
2. go to ```http://localhost:3000/auth/confirm-email```

**Expected behavior**
A title should be present for this route.

**Desktop (please complete the following information):**
 - OS: macOS 12.2.1
 - Browser: chrome
 - Version: 99.0.4844.74
